configure.ac

```diff
@@ -766,6 +763,9 @@ dnl ***************************************************************************
 
 dnl check OpenSSL static linking
 PKG_CHECK_MODULES(OPENSSL, openssl >= $OPENSSL_MIN_VERSION,, OPENSSL_LIBS="")
```

Comment:
I think this is a better solution:
PKG_CHECK_MODULES(OPENSSL, openssl >= $OPENSSL_MIN_VERSION,,)

theoretically it will fail if there is no openssl (see: eventlog/glib/ivykis)